mobile/src/main/java/org/openhab/habdroid/background/ItemsControlsProviderService.kt

```diff
@@ -177,6 +179,12 @@ class ItemsControlsProviderService : ControlsProviderService() {
         }
     }
 
+    private fun createRangeTemplate(item: Item, format: String) = RangeTemplate(
+        item.name, item.minimum?.toFloat() ?: 0F, item.maximum?.toFloat() ?: 100F,
```

Comment:
`Number` items may have a larger value than 100, but no `maximum` defined, e.g. `AlarmSystem_PinCode` on the demo server:
````
AndroidRuntime: FATAL EXCEPTION: DefaultDispatcher-worker-3
    Process: org.openhab.habdroid, PID: 6132
    java.lang.IllegalArgumentException: currentValue=1234.000000 > maxValue=100.000000
        at android.service.controls.templates.RangeTemplate.validate(RangeTemplate.java:181)
        at android.service.controls.templates.RangeTemplate.<init>(RangeTemplate.java:83)
        at org.openhab.habdroid.background.ItemsControlsProviderService.createRangeTemplate(ItemsControlsProviderService.kt:182)
        at org.openhab.habdroid.background.ItemsControlsProviderService.maybeCreateStatefulControl(ItemsControlsProviderService.kt:203)
        at org.openhab.habdroid.background.ItemsControlsProviderService.access$maybeCreateStatefulControl(ItemsControlsProviderService.kt:55)
        at org.openhab.habdroid.background.ItemsControlsProviderService$createPublisherForAllAvailable$job$1.invokeSuspend(ItemsControlsProviderService.kt:100)
        at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
        at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
````